src/login/InitResetForm/InitResetForm.scss

```diff
@@ -0,0 +1 @@
+@import "../../variables.scss";
```

Comment:
What if we didn't add more stylesheets as boilerplate? Maybe we should only add them when we need them?

Reply:
ok no problem with that